atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/SweeperServiceImplTest.java

```diff
@@ -75,73 +94,81 @@ public void sweepingNonExistingTableShouldNotBeSuccessful() {
-
         assertThatExceptionOfType(RemoteException.class)
                 .isThrownBy(() ->
-                        sweeperService.sweepTableFromStartRow(TABLE_REF.getQualifiedName(), INVALID_START_ROW))
-                .matches(ex -> ex.getStatus() == Response.Status.BAD_REQUEST.getStatusCode());
-    }
-
-    @Test
-    public void sweepTableFromStartRowWithBatchConfigWithNullBatchConfigShouldThrow() {
-        setupTaskRunner(Mockito.mock(SweepResults.class));
-
-        when(kvs.getAllTableNames()).thenReturn(ImmutableSet.of(TABLE_REF));
-
-        assertThatExceptionOfType(RemoteException.class)
-                .isThrownBy(() -> sweeperService.sweepTableFromStartRowWithBatchConfig(TABLE_REF.getQualifiedName(),
-                        encodeStartRow(new byte[] {1, 2, 3}), null, null, null))
+                        sweeperService.sweepTableFrom(TABLE_REF.getQualifiedName(), INVALID_START_ROW))
                 .matches(ex -> ex.getStatus() == Response.Status.BAD_REQUEST.getStatusCode());
     }
 
     @Test
     public void sweepTableFromStartRowWithBatchConfigWithNullStartRowShouldBeSuccessful() {
-        setupTaskRunner(Mockito.mock(SweepResults.class));
-
-        when(kvs.getAllTableNames()).thenReturn(ImmutableSet.of(TABLE_REF));
-        sweeperService.sweepTableFromStartRowWithBatchConfig(TABLE_REF.getQualifiedName(), null, 1000, 1000, 500);
+        sweeperService.sweepTable(TABLE_REF.getQualifiedName(), Optional.empty(), Optional.empty(), Optional.of(1000),
+                Optional.of(1000), Optional.of(500));
     }
 
     @Test
     public void sweepTableFromStartRowWithBatchConfigWithExactlyOneNonNullBatchConfigShouldBeSuccessful() {
-        setupTaskRunner(Mockito.mock(SweepResults.class));
-
-        when(kvs.getAllTableNames()).thenReturn(ImmutableSet.of(TABLE_REF));
-        sweeperService.sweepTableFromStartRowWithBatchConfig(TABLE_REF.getQualifiedName(),
-                encodeStartRow(new byte[] {1, 2, 3}), null, 10, null);
+        sweeperService.sweepTable(TABLE_REF.getQualifiedName(),
+                Optional.of(encodeStartRow(new byte[] {1, 2, 3})), Optional.empty(), Optional.of(10), Optional.empty(),
+                Optional.empty());
     }
 
     @Test
     public void testWriteProgressOrPriorityOrMetricsNotUpdatedAfterSweepRunsSuccessfully() {
-        setupTaskRunner(Mockito.mock(SweepResults.class));
-        when(kvs.getAllTableNames()).thenReturn(ImmutableSet.of(TABLE_REF));
-        sweeperService.sweepTableFromStartRow(TABLE_REF.getQualifiedName(), encodeStartRow(new byte[] {1, 2, 3}));
-        Mockito.verify(priorityStore, never()).update(Mockito.any(), Mockito.any(), Mockito.any());
-        Mockito.verify(progressStore, never()).saveProgress(Mockito.any(), Mockito.any());
+        sweeperService.sweepTableFrom(TABLE_REF.getQualifiedName(), encodeStartRow(new byte[] {1, 2, 3}));
+        verify(priorityStore, never()).update(any(), any(), any());
```

Comment:
This should be `verifyNoSweepResultsSaved`.

Reply:
We can also delete this test as we are verifying this for every test anyways, but will not block on this.